src/crontab.ts

```diff
@@ -308,11 +309,24 @@ export const parseCronItem = (
   if (typeof match !== "function") {
     throw new Error("Invalid 'match' configuration");
   }
+
+  let parsedOptions: ParsedCronItemOptions;
+  if (!options) {
+    parsedOptions = { backfillPeriod: 0 };
+  } else if (options.backfillPeriod === undefined) {
+    parsedOptions = { ...options, backfillPeriod: 0 };
+  } else {
+    parsedOptions = {
+      ...options,
+      backfillPeriod: options.backfillPeriod,
+    };
+  }
```

Comment:
I like to avoid the use of `let` where possible because it opens the door to reassignment; please use a ternary instead... Though that's not necessary here:

```suggestion
  const parsedOptions: ParsedCronItemOptions = {
    ...options,
    backfillPeriod: options?.backfillPeriod ?? 0,
  };
```

Reply:
That's embarrassingly much better than my code 😅 Thanks for the suggestion. Applied